R/tt_pos_and_access.R

```diff
@@ -1157,29 +1263,18 @@ setMethod("value_at", "LabelRow",
     }
 }
 
-.copy_titles <- function(new, old) {
-    main_title(new) <- main_title(old)
-    subtitles(new) <- subtitles(old)
-    main_footer(new) <- main_footer(old)
-    prov_footer(new) <- prov_footer(old)
-    new
-}
-
 
 .h_t_body <- function(x, res,
                             keep_topleft,
                             keep_titles,
+                            keep_footers,
                             reindex_refs) {
 
-    if(keep_topleft)
-        top_left(res) <- top_left(x)
-    else
-        top_left(res) <- character()
-    if(keep_titles)
-        res <- .copy_titles(res, x)
-    else
-        ## no titles
-        res <- .copy_titles(res, rtable(" "))
+    res <- .copy_titles_footers_topleft(res, x, 
```

Comment:
This (`.h_t_body`) is so similar to `.copy_titles_footers_topleft` that I'm wondering if we need both functions?

Reply:
This is very true! I did not think I could merge them. Now done